wiki/Tournaments/CMT_4K/2018/en.md

```diff
@@ -61,22 +61,42 @@ The Chinese osu!mania 4K Tournament 2018 is run by various community members by
 
 The following time are based on UTC+08 timezone.
 
-### Group Stage
+### Quarterfinals
```

Comment:
I would use two tables for this, equal to the ones used on world cups, an example
```
## Match Schedule: Quarterfinals

### Saturday, 27. October 2018

| Match Time | Player One |  | Player Two |
| :-: | --: | :-: | :-- |
| 11:00 | zero2snow | vs | ByeForNow |
...
```